src/hooks/useAdjustUrl.ts

```diff
@@ -1,16 +1,37 @@
 import { useEffect } from 'react'
 import { useRouter } from 'next/router'
+import { AppRoutes } from '@/config/routes'
+
+const SAFE_ROUTES = [
```

Comment:
Should we also include routes like `/settings/setup` and `/address-book` here?

Reply:
`/address-book` I deliberately didn't add because it works w/o a Safe.
Will add some of the `/settings/*` 👍